Sources/Tropos/Views/DailyForecastView.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import TroposCore
+
+@objc(TRDailyForecastView) public class DailyForecastView: UIView {
+    @IBOutlet weak var dayOfWeekLabel: UILabel!
+    @IBOutlet weak var highTemperatureLabel: UILabel!
+    @IBOutlet weak var lowTemperatureLabel: UILabel!
+    @IBOutlet weak var conditionsImageView: UIImageView!
```

Comment:
It used to be common to mark outlets as `weak`, but these days it’s recommended to use regular strong references instead.